src/docs/style/illustrations.mdx

```diff
@@ -0,0 +1,73 @@
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Onboarding'}
+        url={'https://material.io/design/communication/onboarding.html'}
+        description={`Material Design's guidelines on onboarding screens.`}
+    />
+    <MaterialDesignDescription
+        title={'Google Marketing Platform Visual Guidelines'}
+        url={
+            'https://designguidelines.withgoogle.com/ads-branding/assets/1hquKP2EcBlDaWyaLkoezqd8r9KPjRgnp/visual-guidelines-google-marketing-platform.pdf#'
+        }
+        description={`You are encouraged to follow their illustration guidelines.`}
+        avatar={<PDFIcon style={{ width: 40, height: 40 }} />}
+    />
+</React.Fragment>
+
+## Designing the Illustration
+
+### Perspective
+
+Your illustration should be either in a horizontal perspective (neither looking up nor looking down) or in parallel projection.
+
+### Ratio
+
+The width-height ratio should be between 1:1 to 2:3, which gives a margin on a mobile phone for UI interactions. If designing specifically for desktop applications (that is, the illustration will never show up on mobile), you can go wider than that, but no more than 16:9.
+
+### Drawing Style
+
+Keep your style consistent within one application, including your illustrations for the webpage for the product. Do not be too abstract or artistic, as your main goal is to inform your users of certain features in your product, instead of decorating it.
+
+### Text
+
+Avoid the use of text, as they might require translations, and can be hard to see on mobile. If you have to, use Open Sans, just like other PX Blue applications, and ensure that they are legit on the smallest targeted device.
+
+### Color
+
+Use [PX Blue palette](/style/color) as the base color. If there is a gradient, make sure the two ends of the gradient are using PX Blue colors. Use some blue hue (this can be colors from primary blue, Eaton blue, or light blue) somewhere in the foreground for your drawing.
+
+Avoid using colors that may blend into the app theme background (suggest adding a background color to pad around it). Avoid using the 500 colors from the primary color palette, as they may blend with other UI elements (buttons, progressors).
+
+Avoid too many dark strokes for light themes, or light strokes for dark themes. The illustrations might be reused as infographics, and line strokes might intervene with labels.
+
+Test your illustration in both light theme (white 50) and dark theme (black 900). It is recommended that you modify your design for light backgrounds and dark backgrounds.
```

Comment:
```suggestion
Test your illustrations on both light (white 50) and dark (black 900) application backgrounds. If your application will support multiple themes, we recommended having a version of your icons for each.
```

Reply:
Modified to "version of your **illustrations** for each"